tests/test_lag_2.py

```diff
@@ -119,6 +128,62 @@ def check_single_lag(common_setup_teardown, nbrhosts, lag_name):
     vm_host = nbrhosts[peer_device]
     verify_lag_minlink(duthost, vm_host, lag_name, peer_device, intf, neighbor_interface, po_interfaces, po_flap, deselect_time=95)
 
+def check_lag_fallback(common_setup_teardown, nbrhosts, lag_name):
+    duthost, ptfhost, vm_neighbors, mg_facts, lag_facts, fanout_neighbors = common_setup_teardown
+    logging.info("Start checking lag fall back for: %s" % lag_name)
+    intf, po_interfaces = get_lag_intfs(lag_facts, lag_name)
+    po_fallback = lag_facts['lags'][lag_name]['po_config']['runner']['fallback']
+
+    # Figure out remote VM and interface info for the lag member and run lag fallback test
+    peer_device = vm_neighbors[intf]['name']
+    neighbor_interface = vm_neighbors[intf]['port']
+    vm_host = nbrhosts[peer_device]
+
+    try:
+        # Shut down neighbor interface
+        set_neighbor_interface(vm_host, neighbor_interface, shut=True)
```

Comment:
Can we double check the neighbor interfaces status after this shutdown cmd?

Just want to make sure the neighbor interface is actually down to avoid false alarm. Thanks

Reply:
Our AnsibleHostBase will raise an Exception if the ansible module invoking returns failure(although it will returns 'changed': true/false).
But I found that the ansible eos module not always returns a clear failure which will cause the state change is not guaranteed.

I think it is better to check the state after shutdown the interface.
set_neighbor_interface method has been extract to a class method in EOSHost in another PR. I will refactor this part and guaranteed the operation is succeed in that method.

Reply:
Sounds good. Thanks Blueve!